src/iitkgp_erp_login/test.py

```diff
@@ -0,0 +1,17 @@
+import erp 
+import requests
+
+headers = {
+   'timeout': '20',
+   'User-Agent': 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Ubuntu Chromium/51.0.2704.79 Chrome/51.0.2704.79 Safari/537.36',
+}
+
+session = requests.Session()
+
+
+try :
+  erp.login(headers,session, LOGGING=True)
+
+except Exception as e:
+    print("Error :"+ str(e))
+      
```

Comment:
why is this here :harold:
If you wanna add tests, add them in new folder in the root of the project inside tests folder.
Also do this testing implementation in a separate PR.

Reply:
Will remove it

Reply:
how do I use the package for develpoment

Reply:
Traceback (most recent call last):
  File "/home/unc-128/Projects/iitkgp-erp-login-pypi/src/iitkgp_erp_login/test.py", line 12, in <module>
    erp.login(headers, session) # This will prompt the user to enter their roll number, password, and OTP.
  File "/home/unc-128/Projects/iitkgp-erp-login-pypi/src/iitkgp_erp_login/erp.py", line 192, in login
    secret_question = get_secret_question(headers=headers, session=session, roll_number=ROLL_NUMBER, log=LOGGING)
  File "/home/unc-128/Projects/iitkgp-erp-login-pypi/src/iitkgp_erp_login/erp.py", line 70, in get_secret_question
    raise ErpLoginError("Invalid Roll Number",log)
erp.ErpLoginError: Invalid Roll Number

Reply:
> how do I use the package for develpoment
https://github.com/proffapt/iitkgp-erp-login-pypi/blob/f5d3465dc0af8166e8229520dd2e62b0d102a2d6/.gitignore#L14

For now I was keeping the tests in this folder which is in git ignore. You can do the same for the time being and when there will be more tests added then we can remove it from .gitignore file.

Reply:
how do I import erp here